### ISSUE TITLE
[Bug] AA vehicle missing delay before starting to drive

### ISSUE THREAD

**Issue by Bob-Murphy** [official-antistasi-community/A3-Antistasi#3074]
When an airfield is spawned in, the AA vehicle starts driving around immediately.
As the AA vehicle is getting spawned as one of the first assets, it can happen that into spots where other vehicles or assets spawn, which then leads to shit blowing up.

The AA vehicle starting to drive should have a 10 second delay at least to be on the save side as spawning time duration of other assets can vary.

![ArmA 3 Screenshot 2023 11 28 - 16 51 34 89](https://github.com/official-antistasi-community/A3-Antistasi/assets/50651578/63efbc04-4ae6-416e-9ccb-138be01cab73)
![ArmA 3 Screenshot 2023 11 28 - 16 51 36 45](https://github.com/official-antistasi-community/A3-Antistasi/assets/50651578/99a0af38-a358-43d6-9725-5c3cde74268c)
![ArmA 3 Screenshot 2023 11 28 - 16 51 42 84](https://github.com/official-antistasi-community/A3-Antistasi/assets/50651578/1fc68646-a4b8-407e-aec0-0a19173f4ce4)

**Comment by ante185** [comment]
this is a duplicate of #2880, so when it comes time to close this issue go and close that one too.
